pkg/gateways/store/postgres/accounts/upsert.go

```diff
@@ -0,0 +1,15 @@
+package accounts
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/daniel1sender/Desafio-API/pkg/domain/entities"
+)
+
+func (ar AccountRepository) Upsert(account entities.Account) error {
+	if _, err := ar.Exec(context.Background(), "INSERT INTO accounts(id, name, cpf, secret, balance, created_at) VALUES ($1, $2, $3, $4, $5, $6) ON CONFLICT (id) DO UPDATE SET balance = EXCLUDED.balance", account.ID, account.Name, account.CPF, account.Secret, account.Balance, account.CreatedAt); err != nil {
```

Comment:
```suggestion
        statement := `INSERT INTO accounts(
    id,
    name,
    cpf,
    secret,
    balance,
    created_at
) VALUES (
    $1,
    $2,
    $3,
    $4,
    $5,
    $6
) ON CONFLICT (id) DO UPDATE SET balance = EXCLUDED.balance`
	if _, err := ar.Exec(context.Background(), statement, account.ID, account.Name, account.CPF, account.Secret, account.Balance, account.CreatedAt); err != nil {
```
well, it can be clearer, but this idea of extracting and formatting helps to enhance the readability of your code. Avoid long lines with go multiline string syntax....you could also extract the statement as a top-level constant if you wish